style.css

```diff
@@ -571,4 +607,113 @@ h5 {
   .latest-img {
     flex: 2;
   }
+
+  .row-deals {
+    flex-direction: column;
+    margin-bottom: 10px;
+    justify-content: flex-start;
+  }
+
+  .item-deals {
+    padding: 5px;
+    text-align: center;
+  }
+}
+
+@media (max-width: 768px) {
+  nav {
+    flex-wrap: wrap;
+    font-size: 0.8rem;
+  }
+
+  .nav-top {
+    align-items: center;
+    justify-content: center;
+    padding: 5px;
+  }
+
+  .nav-bottom {
+    font-size: 0.7rem;
+    padding: 5px 2px 20px 2px;
+    justify-content: center;
+    flex-direction: row;
+    flex-wrap: nowrap;
+  }
+
+  .li-bottom {
+    margin-right: 5px;
+    font-size: 0.7rem;
+  }
+
+  .nav-bottom-right {
+    border: 1px solid black;
+    padding: 3px;
+  }
+
+  .container {
+    width: 90%;
+  }
```

Comment:
## `Latest News`, `Footer` && `Categories` - Screen Change
- [ ] The Change in the `Latest News`, `Footer` && `Categories` sections requires that the articles get to span the entire page. On the clone page, 👇 

#### `Latest News`
|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone page](https://raw.githack.com/MarkoNS1990/the_next_web/milestone-2/index.html)|
|:--:|:--:|
|![latest_deals_small_screen](https://user-images.githubusercontent.com/22345961/109030356-c7133080-76d4-11eb-918d-09351087dcbe.png)|![latest_deals_small_screen_clone](https://user-images.githubusercontent.com/22345961/109030379-cbd7e480-76d4-11eb-994e-b900487a2f9e.png)|

##

#### `Categories`
|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone page](https://raw.githack.com/MarkoNS1990/the_next_web/milestone-2/index.html)|
|:--:|:--:|
|![categories_small_screen](https://user-images.githubusercontent.com/22345961/109030105-93d0a180-76d4-11eb-86eb-1ce60ba29f2e.png)|![categories_small_screen_clone](https://user-images.githubusercontent.com/22345961/109030143-9cc17300-76d4-11eb-90db-4a9cfb6cec01.png)|
##

#### `Footer`
|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone page](https://raw.githack.com/MarkoNS1990/the_next_web/milestone-2/index.html)|
|:--:|:--:|
|![footer_small_screen](https://user-images.githubusercontent.com/22345961/109030215-aba82580-76d4-11eb-90dd-e754279e7fea.png)|![footer_small_screen_clone](https://user-images.githubusercontent.com/22345961/109030235-b06cd980-76d4-11eb-971e-0a334925ed6f.png)|